src/count/origami.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
Again this can probably be removed and move the relevant parts (demos) to the o-comments origami file.